bylaws.html

```diff
@@ -11,99 +11,99 @@ <h1>CouchDB Bylaws</h1>
+<p>Roles are never rescinded because of inactivity—unless that inactivity is causing a problem for the project. Fortunately, our decision-making process means that inactivity is very rarely a problem. Roles will be rescinded if the <a href="#pmc">Project Management Committee</a> (or <a href="#pmc">PMC</a>, see 2.4. below) believes the individual is no longer able to discharge the duty of the role responsibly.
 
-<p>We understand that you have many roles in life. We use a hat metaphor to talk about these roles. For instance, you might have your work hat as well as several ASF hats. We expect you to know when to wear the appropriate hat, and to comport yourself in a manner befitting the interests of the Foundation when interacting with the project. Failure to do this is a serious dereliction of duty.
+<p>We understand that you have many roles in life. We use a hat metaphor to talk about these roles. For instance, you might have your work hat as well as several ASF hats. We expect you to know when to wear the appropriate hat, and to comport yourself in a manner befitting the interests of the foundation when interacting with the project. Failure to do this is considered a severe dereliction of duty.
 
-<p>Sometimes it is a good idea to tell people which hat you are wearing. For instance, the PMC Chair might start an informal email by stating they are not wearing the PMC Chair's hat, just to be clear about how the statements ought to be interpreted.
+<p>Sometimes it is a good idea to tell people which hat you are wearing. For instance, the PMC chair might start an informal email by explicitly stating that they are not wearing the PMC chair's hat. Doing so can help to clear up the meaning (and implication) of what is said.
 
-<p>We expect you to act in good faith. This is very important for a community that depends so heavily on trust.
+<p>We expect you to act in good faith. Doing so is very important for a community that depends so heavily on trust.
 
 <h3 id="users">2.1. Users</h3>
 
-<p><strong>The most important participants in the project are people who use our software.</strong>
+<p><strong>The most critical participants in the project are people who use our software.</strong>
 
-<p>Users can participate by talking about the project, providing feedback, and helping others. This can be done at the ASF or elsewhere, and includes being active on <a href="https://lists.apache.org/list.html?user@couchdb.apache.org">the user mailing list</a>, third-party support forums, blogs, and social media. Users who participate in this way automatically become contributors.
+<p>Users can participate by talking about the project, providing feedback, and helping others. Participation can be done at the ASF or elsewhere and includes being active on the user mailing list, third-party support forums, blogs, and social media. Users who participate in this way become contributors automatically.
```

Comment:
-1 on removing the link to the mailing list. Why would you make it harder to navigate to our community from the bylaws?